cli/ballerina-cli/src/main/resources/cli-help/ballerina-test.help

```diff
@@ -125,3 +130,7 @@ EXAMPLES
        Run the test functions with the code coverage and generate a coverage
        report in the XML format.
            $ bal test --test-report --code-coverage --coverage-format=xml
+
+       Run the tests with the code coverage and exclude files from coverage
+       calculation.
+           $ bal test --code-coverage --excludes=<source_path1>,<source_path2>
```

Comment:
What if the user wants to exclude a complete module? Should the user give the individual paths of each file of that module?

Reply:
Do we need to support module basis or a directory basis? (If the user wants to exclude only the generated sources of a specific module (module has both generated and non-generated sources). Then excluding by module directory may not be useful. For this kind of scenarios, specifying the path of the directory is useful.)
For an instance,
`./` - All the sources of default module.
`./modules/<module-name>` - All the sources under that module.
`./generated/<module-name>` - All the generated sources under that specific module